src/main/java/com/artipie/rpm/CliArguments.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * Cli tool argument parsing.
+ *
+ * @since 0.9
+ * @todo #89:30min Cli options are not parsed correctly. getOptionValue method always return
```

Comment:
@victornoel parsing should work when you apply my comments

Reply:
@Vatavuk I don't see any reason to do that because:
- first, we want `-d` or `-n` to work, so if I use here `dgst` (or `np` below) it will obviously break the CLI, which we don't want
- second, the rest of the code actually uses `DIGEST.getOpt()` (and `NAMING_POLICY.getOpt()`) so I suppose this todo is theoretically fixed. I haven't tried.

To be honest, the todo is not assigned to me, I believe the person who will solve it will add tests and formally verify it is fixed and all will be well. I don't see any reason to do more work here with respect to the issue I'm currently solving and it will be good for the project to let the current assignee of this todo do his work.

Reply:
@victornoel my bad, I've confused arg name with the option. You can remove the puzzle then, it's working I've tried. The second puzzle will test it further.

Reply:
@Vatavuk which second puzzle? The one I introduced?

Reply:
@Vatavuk maybe you didn't realise this is an existing todo that was moved? I haven't introduced it

Reply:
@Vatavuk it is already assigned to someone, see #113

Reply:
@victornoel we can leave it then.